src/utils/index.js

```
@@ -188,10 +188,17 @@ export const setToLocalStorage = (key, value) =>
 export const removeFromLocalStorage = key => localStorage.removeItem(key);
 
 export const getFromLocalStorage = (key, defaultValue) => {
-  // eslint-disable-next-line @bigbinary/neeto/no-local-storage
-  const storedValue = localStorage.getItem(key);
+  try {
+    // eslint-disable-next-line @bigbinary/neeto/no-local-storage
+    const storedValue = localStorage.getItem(key);
+
+    return storedValue ? JSON.parse(storedValue) : defaultValue;
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(error);
```

We don't need to log the error to the console, silently return the default value.

---

This is completed in https://github.com/bigbinary/neeto-ui/pull/2420/commits/d58872ceb89f4c0602724d113f3037f919cefb92.